### ISSUE TITLE
Панель слайдера

### ISSUE THREAD

**Issue by EvgeniyChuprov** [fatfutor/range-slider#5]
Панель настроек слайдера, не должны быть часть слайдера.
из ризомы:
  - возможность на лету изменить значение "снаружи" javascript-ом, 
 - Рядом с каждым слайдером разместить панель конфигурирования, чтобы можно было на лету менять параметры.

**Comment by fatfutor** [comment]
В коде есть класс Panel - это не слайдер. Не знаю насколько я корректно реализовал. я импортировал Presenter в Panel и в source/index вместо Presenter создаю экземпляр Panel

**Comment by EvgeniyChuprov** [comment]
Суть плагина в том, что это готовое решение, с минимально необходимым интерфейсом для пользователя.  Сейчас же получается, что пользователю придется работать не с внешним интерфейсом, а копаться во внутренностях самого плагина и что то там делать.

**Comment by EvgeniyChuprov** [comment]
Эти инпуты должны относится к панели, сейчас как я понял они часть слайдера? 
![Снимок экрана от 2020-02-11 13-59-45](https://user-images.githubusercontent.com/36810364/74231147-ed874900-4cd6-11ea-98d1-5b3607e7c858.png)
в панели я их не вижу
![Снимок экрана от 2020-02-11 14-02-37](https://user-images.githubusercontent.com/36810364/74231245-39d28900-4cd7-11ea-8e28-95cc93a54dd2.png)

**Comment by fatfutor** [comment]
это часть слайдера, у слайдера ведь должны быть инпуты введенных данных чтобы по кнопке submit отправить на сервер, текст выше "от до" - это просто текст, он показывает не данные, а размер всего диапазона, и не положение ползунков

**Comment by EvgeniyChuprov** [comment]
Нет. Инпуты должны быть частью панели. Слайдеру должно быть без разницы с панелью данными обмениваться или с сервером. Он должен отдавать объект настроек и ему все равно кто его будет принимать, с получением данных тоже самое.

**Comment by fatfutor** [comment]
Ступор. не получается. Изменения с инпутов панели идут в слайдер, а обратно только если инпуты панели не трогать и то как-то криво. не знаю почему слетают обработчики событий с ползунков после активации инпутов с панели
